src/views/StandaloneDirectorsFiling.vue

```diff
@@ -632,20 +645,42 @@ export default {
       }
     },
 
-    toggleFiling (setting, filing) {
-      let added = false
-      for (let i = 0; i < this.filingData.length; i++) {
-        if (this.filingData[i].filingTypeCode === filing) {
-          if (setting === 'add') {
-            added = true
-          } else {
-            this.filingData.splice(i, 1)
-          }
-          break
+    /**
+     * Adds/removes codes or sets flags in the Filing Data object.
+     * @param addRemove Whether to add or remove the specified codes/flags.
+     * @param filingCode The Filing Type Code to add or remove (optional).
+     * @param priority The Priority flag to set or clear (optional).
+     * @param waiveFees The Waive Fees flag to set or clear (optional).
+     * @example setFilingData('add', undefined, undefined, true) -> adds Waive Fees to all codes
+     * @example setFilingData('remove', undefined, true, undefined) - removes Priority from all codes
+     * @example setFilingData('add', 'OTFDR', false, false) -> adds OTFDR code with neither flag
+     * @example setFilingData('add', 'OTCDR') -> adds OTCDR code with default flags
```

Comment:
Extensive examples!

Reply:
The functionality is the same in all components but I tailored the examples :)
This is a candidate for a mixin, for sure. Maybe in my next commit :)

Reply:
Sounds good to me!